### PR TITLE
Switch to pyproject.toml

### PR DESCRIPTION
**Pull request by dmulcahey**
## REQUIRES: https://github.com/zigpy/workflows/pull/2

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+    "pytest>=7.1.2",
+    "pytest-asyncio>=0.19.0",
+    "pytest-timeout>=2.1.0",
+    "pytest-mock>=3.8.2",
+    "pytest-cov>=3.0.0",
+]
+
+[tool.isort]
+profile = "black"
+not_skip = "_init__.py"
```
Comment (TheJulianJES) [inline pyproject.toml:36]: One underscore missing?

[the remark this answers is not in the document]
Comment (dmulcahey) [reply pyproject.toml:36]: oops

**Comment by dmulcahey** [comment]
This needs to be rerun once https://github.com/zigpy/workflows/pull/1 is merged

**Comment by TheJulianJES** [comment]
Superseded by:
- https://github.com/zigpy/zha-device-handlers/pull/2491

**Pull request closed without merging**